src/styling/Home.scss

```diff
@@ -0,0 +1,77 @@
+@import url(http://fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300ita‌​lic,400italic,500,500italic,700,700italic,900italic,900);
+
+$color: #4369b2;
+
+body {
+  background: $color;
+  margin: 0;
+  font-family: 'Roboto', sans-serif;
```

Comment:
* Kindly follow the design guidelines which include font families( Lato, Gill sans) as its [required](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=You%20should%20follow,space%20between%20elements.)

Reply:
Alright Thank you for the correction